### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by zelongc**
Add description of the difference between multi doge and dogecoin core. As a supplement to what's already on https://dogecoin.com/

### CODE REVIEW

File: doc/FAQ.md [hunk 1]
```diff
@@ -52,3 +52,9 @@ four-hour difficulty readjustment:**
 | 400,000–499,999      |           0–62,500 |
 | 500,000–599,999      |           0–31,250 |
 | 600,000+             |             10,000 |
+
+### what's the difference bertween dogecoin core and multi-doge?
```
[the remark this answers is not in the document]
Comment (Lizxxx) [reply doc/FAQ.md:56]: Lol let go

[the remark this answers is not in the document]
Comment (Lizxxx) [reply doc/FAQ.md:56]: 54222122187

**Review by Lizxxx — APPROVED** [review]

File: doc/FAQ.md (hunk 1 above)
Comment (chromatic) [inline doc/FAQ.md:56]: Nit: capitalize "what" and change to "between"

File: doc/FAQ.md [hunk 2]
```diff
@@ -52,3 +52,9 @@ four-hour difficulty readjustment:**
...
+
+Core: The go to standard, can do anything and everything. By running core you are running a dogecoin node (if you forward port 22556 on your router) and contributing to the decentralized consensus mechanism that makes dogecoin work. You don't need to trust others that incoming transactions are legit because you verify them yourself. Takes a lot of bandwidth, memory, and hard drive space to run.
```
Comment (chromatic) [inline doc/FAQ.md:58]: I think this is largely true and accurate. This description doesn't mention that you can run Core with or without a wallet. I'm not sure if it's too much detail to add though.

**Comment by patricklodder** [comment]
I have 2 problems with this one in general:

1. Multidoge is not maintained by this org, so it's an external implementation to Dogecoin Core. I think that we shouldn't comment on any external implementations of the protocol in this repository's code or docs, unless it's essential to the protocol.
2. Multidoge has not seen any development since 2016

I think it'd be better that the owner of the website deal with this type of issues than us. I do not know who currently owns the domain - perhaps @rnicoll can clarify, so that we can direct this issue to the proper channel?

**Comment by chromatic** [comment]
Fair points. @zelongc can you share more about your motivation for this change? If it's to add information similar to what's on dogecoin.com then we should consider whether the latter is up to date.

If it's to answer a question like "What's the purpose of Dogecoin Core and why would or wouldn't I use it?" then that's a different question. It's not answered in the README, so either here or there might be the right place.

**Comment by patricklodder** [comment]
@zelongc,

I've just received an answer on this. The new owner of the dogecoin.com website seems to be one of @dogecoinfoundation's legal entities according to a [tweet](https://twitter.com/jwiechers/status/1530950645947240448) by @jwiechers. I believe they have been working on a new website at [dogecoinfoundation/dogecoin.com](https://github.com/dogecoinfoundation/dogecoin.com). It would be better to direct this information there, because then it will be get much more visibility!

Thank you for caring about this though! It's appreciated!

**Pull request closed without merging**